python/tchannel/zipkin/tracers.py

```diff
@@ -114,10 +118,13 @@ def __init__(self, logger):
     def record(self, traces):
         for (trace, annotations) in traces:
             self._logger.info(
-                base64_thrift_formatter(trace, annotations)
+                thrift_formatter(trace, annotations, isbased64=True)
             )
 
 
+TCollectorClient = client_for('tcollector', TCollector)
```

Comment:
I wonder if we want to be able to provide different service names. Like, if there are different implementations of the service in different datacenters, the user may want to provide different names.

Don't do anything about that just yet. This may require a change on the Thrift client implementation side.

Can you add a TODO here?